src/components/harvestLogs/HarvestLogDrawer.tsx

```diff
@@ -322,32 +338,50 @@ const HarvestLogDrawer = ({
         <InputLabel htmlFor="harvest-log-product">Product - Unit</InputLabel>
         <OutlinedInput
           id="harvest-log-product"
-          value={selectedProduct}
+          value={season?.product?.name ?? "-"}
           disabled
         />
       </Box>
 
       <Controller
         control={control}
         name="collectedAmount"
-        render={({ field }) => {
+        render={({ field: { ref, value, onChange } }) => {
           return (
             <Box display="flex" flexDirection="column" gap={1}>
               <InputLabel htmlFor="harvest-log-amount">Amount*</InputLabel>
-              <OutlinedInput {...field} id="harvest-log-amount" />
+
+              <TextField
+                id="harvest-log-amount"
+                ref={ref}
+                value={value}
+                onChange={(event) => {
+                  if (event.target.value) {
+                    onChange(Number(event.target.value));
+                  } else {
+                    onChange(undefined);
+                  }
+                }}
+                margin="dense"
+                type="number"
+                error={!!errors.collectedAmount}
+                helperText={errors.collectedAmount?.message}
+                variant="outlined"
+              />
             </Box>
           );
         }}
       />
       <Controller
         control={control}
-        name="seasonDeductionsIds"
-        render={({ field }) => {
+        name="seasonDeductionIds"
+        render={() => {
           return (
             <Box display="flex" flexDirection="column" gap={1}>
               <InputLabel id="harvest-log-deductions">Deduction</InputLabel>
               <Select
-                {...field}
+                // {...field}
```

Comment:
remove this comments

Reply:
https://github.com/LangaraVoyagers/project-2/pull/22/commits/37bb5e2b705b815a472430f72fd13585aadc1c33